src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -308,10 +310,68 @@ export default class BrowseSkill extends React.Component {
             return result;
           });
         }
-        let cardsToDisplay = data.filteredData.length;
-        cardsToDisplay = cardsToDisplay < 10 ? cardsToDisplay : 10;
         self.setState({
-          topRatedSkills: data.filteredData.slice(0, cardsToDisplay),
+          topRatedSkills: data.filteredData,
+        });
+      },
+    });
+  };
+
+  loadTopUsed = () => {
+    let url;
+    url =
+      urls.API_URL +
+      '/cms/getSkillList.json?group=All&applyFilter=true&filter_name=descending&filter_type=usage&count=10';
+    let self = this;
+    $.ajax({
+      url: url,
+      dataType: 'jsonp',
+      jsonp: 'callback',
+      crossDomain: true,
+      success: function(data) {
+        if (self.state.searchQuery.length > 0) {
```

Comment:
@akshatnitd you can remove the search query filter. :+1:

Reply:
Yes, doing it.